main.nf

```diff
@@ -657,17 +725,25 @@ process index_chromatograms {
 }
 
 
+// Combine osw files and chromatograms by experimental design condition
```

Comment:
don't know if I understand this comment. What is done here?

Reply:
ok, ill try to make it more clear that is basically just a step reformatting the DIAlignR output to fit to msstats by combining it with the experimental design and spectral library table.